Resources/config/command.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sonata.core.command.dump_doctrine_metadata" class="Sonata\CoreBundle\Command\SonataDumpDoctrineMetaCommand">
```

Comment:
No alias?

Reply:
Which alias do you mean? Command name for tag or alias for service? And for what?

Reply:
Thought about [lazy command](https://symfony.com/blog/new-in-symfony-3-4-lazy-commands) aliases. But this is only possible in symfony 3.4+